plugins/gatsby-source-data/main.js

```diff
@@ -37,8 +36,8 @@ const dataExists = name => {
 const getLiveSessionsFromSheets = async apiKey => {
   const sheets = new Sheets(apiKey);
   const values = await sheets.getLiveSessions();
-  values.shift() // Get rid of the first row which is just the title
-  const keys = values.shift();
+  values.shift().shift() // Get rid of the first row which is just the title
+  const keys = ['Date', 'Time', 'Mosque',	'Title', 'Speaker', 'Link']
```

Comment:
_nitpick_
Use map instead of double for loops. Why?
1. The relationship between keys and sessions is not very clear
2. 2 for loops: It doesn't really matter given the length of the array. However, readability of the code is affected.

The intention can be better reflected if we can be more explicit:
```
return liveSessions.map(session => (
    {
        Date: session[0].split(', ')[1],
        Time: session[1],
        Mosque: session[2],
        Title: session[3],
        Speaker: session[4],
        Link: session[5],
    }
))
```